From e3facc69d13672ffc4dd39fbccfaac790a2041c3 Mon Sep 17 00:00:00 2001
From: Deepali Chourasia <deepch23@in.ibm.com>
Date: Tue, 10 May 2022 05:20:04 -0400
Subject: [PATCH] disable LIBOMPTARGET to fix build on baremetal

---
 recipe/build-llvm-openmp.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/recipe/build-llvm-openmp.sh b/recipe/build-llvm-openmp.sh
index 7ccd373..3c69b9d 100644
--- a/recipe/build-llvm-openmp.sh
+++ b/recipe/build-llvm-openmp.sh
@@ -17,6 +17,8 @@ fi
 
 cmake \
     -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DENABLE_LIBOMPTARGET=OFF \
+    -DOPENMP_ENABLE_LIBOMPTARGET=OFF \
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_PREFIX_PATH=$PREFIX \
     ..
-- 
2.23.0

