From 9e92f103c71435c079901f58a4dc182283941fff Mon Sep 17 00:00:00 2001
From: Deepali Chourasia <deepch23@in.ibm.com>
Date: Tue, 7 Jun 2022 11:56:05 +0000
Subject: [PATCH] disable LIBOMPTARGET to fix build on baremetal

---
 recipe/build-llvm-openmp.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/recipe/build-llvm-openmp.sh b/recipe/build-llvm-openmp.sh
index ede66f5..d34c75d 100644
--- a/recipe/build-llvm-openmp.sh
+++ b/recipe/build-llvm-openmp.sh
@@ -16,6 +16,8 @@ fi
 
 cmake ${CMAKE_ARGS} \
     -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DENABLE_LIBOMPTARGET=OFF \
+    -DOPENMP_ENABLE_LIBOMPTARGET=OFF \
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_PREFIX_PATH=$PREFIX \
     ..
-- 
2.34.1

