From 1978047b0c3613ff84e874a147eb4d5f1810af0d Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Fri, 20 Nov 2020 09:59:18 +0000
Subject: [PATCH] Fix test

---
 recipe/SSLTest.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/recipe/SSLTest.cmake b/recipe/SSLTest.cmake
index 1fd943a..23f9e2d 100644
--- a/recipe/SSLTest.cmake
+++ b/recipe/SSLTest.cmake
@@ -1,5 +1,5 @@
 set(FILE_NAME "LICENSE.txt")
-set(DOWNLOAD_URL "https://raw.githubusercontent.com/conda-forge/cmake-feedstock/master/${FILE_NAME}")
+set(DOWNLOAD_URL "https://raw.githubusercontent.com/conda-forge/cmake-feedstock/26e3ecb4156c14d90a66fd1433d52a1d7e27946d/${FILE_NAME}")
 set(EXPECTED_SHA256 "344c439ba60e5db75f0c894fdd22e3445db089b3bd936806bdf128c0016c12c7")
 
 file(DOWNLOAD ${DOWNLOAD_URL} ${CMAKE_CURRENT_BINARY_DIR}/${FILE_NAME}
-- 
1.8.3.1

