From 79e936a9c74711090ef8bca49e8bddadef0dc372 Mon Sep 17 00:00:00 2001
From: Deepali Chourasia <deepch23@in.ibm.com>
Date: Mon, 23 May 2022 13:02:01 +0000
Subject: [PATCH] fix build with GCC11

---
 recipe/build.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/recipe/build.sh b/recipe/build.sh
index 0df298f..a53da52 100644
--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -7,6 +7,9 @@ sed -i.bak "s/libLTO.dylib/libLTO.${PKG_VER_ARRAY[0]}.dylib/g" lib/Driver/ToolCh
 mkdir build
 cd build
 
+export CXXFLAGS="${CXXFLAGS} -fplt"
+export CFLAGS="${CFLAGS} -fplt"
+
 if [[ "$variant" == "hcc" ]]; then
   EXTRA_ARGS="$EXTRA_ARGS -DKALMAR_BACKEND=HCC_BACKEND_AMDGPU -DHCC_VERSION_STRING=2.7-19365-24e69cd8-24e69cd8-24e69cd8"
   EXTRA_ARGS="$EXTRA_ARGS -DHCC_VERSION_MAJOR=2 -DHCC_VERSION_MINOR=7 -DHCC_VERSION_PATCH=19365"
-- 
2.34.1

