diff --git a/recipe/build.sh b/recipe/build.sh
index 9de6fd3..faf9010 100755
--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -86,7 +86,8 @@ cmake ${CMAKE_ARGS} ..  \
       -DCMAKE_AR=${AR} \
       -DCMAKE_RANLIB=${RANLIB} \
       -DCMAKE_VERBOSE_MAKEFILE=ON \
-      -DProtobuf_PROTOC_EXECUTABLE=$BUILD_PREFIX/bin/protoc
+      -DProtobuf_PROTOC_EXECUTABLE=$BUILD_PREFIX/bin/protoc \
+      -DCMAKE_CXX_STANDARD=17   # this is for std=c++17
 
 ninja install -v
 
